tests/weary_test.cpp

```diff
@@ -66,23 +66,23 @@ TEST_CASE( "weary_assorted_tasks", "[weary][activities]" )
         INFO( info.summarize() );
         INFO( guy.debug_weary_info() );
         REQUIRE( !info.empty() );
-        CHECK( info.transition_minutes( 0, 1, 120_minutes ) == Approx( 120 ).margin( 5 ) );
-        CHECK( info.transition_minutes( 1, 2, 250_minutes ) == Approx( 250 ).margin( 5 ) );
-        CHECK( info.transition_minutes( 2, 3, 360_minutes ) == Approx( 360 ).margin( 5 ) );
-        CHECK( info.transition_minutes( 3, 4, 465_minutes ) == Approx( 465 ).margin( 5 ) );
+        CHECK( info.transition_minutes( 0, 1, 120_minutes ) == Approx( 115 ).margin( 0 ) );
+        CHECK( info.transition_minutes( 1, 2, 250_minutes ) == Approx( 255 ).margin( 0 ) );
+        CHECK( info.transition_minutes( 2, 3, 360_minutes ) == Approx( 360 ).margin( 0 ) );
+        CHECK( info.transition_minutes( 3, 4, 465_minutes ) == Approx( 470 ).margin( 0 ) );
```

Comment:
```suggestion
        CHECK( info.transition_minutes( 0, 1, 115_minutes ) == Approx( 115 ).margin( 0 ) );
        CHECK( info.transition_minutes( 1, 2, 255_minutes ) == Approx( 255 ).margin( 0 ) );
        CHECK( info.transition_minutes( 2, 3, 360_minutes ) == Approx( 360 ).margin( 0 ) );
        CHECK( info.transition_minutes( 3, 4, 470_minutes ) == Approx( 470 ).margin( 0 ) );
```
`info.transition_minutes` checks for the transition (of the given type) closest to the time input.